server/src/app.module.ts

```diff
@@ -12,11 +13,13 @@ import {
 } from './modules';
 
 import config from './core/database/connection';
-
 @Module({
   imports: [
     SequelizeModule.forRoot({
       ...config,
+      sync: { force: false },
+      autoLoadModels: true,
+      synchronize: true,
       models: [],
```

Comment:
call the models here

Reply:
I used an autoload model with synchronize so you don't have to put them in the array of models.
But I will delete the model property because it becomes useless

Reply:
well done!